Appraisals

```diff
@@ -55,3 +55,10 @@ appraise "rails71" do
   gem "pg", ">= 1.1"
   gem "sqlite3", ">= 1.4"
 end
+
+appraise "rails80" do
+  gem "rails", ">= 7.1.0.beta1", "< 8.1"
```

Comment:
@danielmorrison @fernandomenolli 

This is not rails 8. Rails 8 is the `main` branch which is `8.0.0.alpha` which is **NOT** released on rubygems.

this will target at best `7.2.x`